CONTRIBUTING.md

```diff
@@ -79,13 +79,35 @@ For example:
 
 Over time, we should track which items are useful and which don't get any traffic. Also, as Apify Docs expand, we may need to add more cards and update which articles we link to.
 
-## Local setup
+## Installation and setup
 
-If you want to work only on the main documentation content, cloning this repository is enough. Once you run `npm install` to install all dependencies and run `npm start`, the main portal will open on <http://localhost:3000>. All the links in the navbar and footer need to be absolute, and they will use a different hostname, configured to `docs.apify.loc` - to use that, follow the steps below and set up the Nginx server.
+1. Use Git to clone this repository.
+2. If you work at Apify and can install packages prefixed with `@apify-packages`, you can skip this step.
+   Otherwise go to GitHub and navigate to [Settings / Personal access tokens / Tokens (classic)](https://github.com/settings/tokens).
+   Despite the UI suggesting you should use the new fine-grained tokens, generate a **classic token** with scopes **repo** and **write:packages**.
+   Keep the token somewhere safe.
+   Then run the following command and use your GitHub username together with the token as credentials:
 
-Alternatively, you can skip the Nginx part and navigate to <http://localhost:3000/academy> or <http://localhost:3000/platform> manually instead of using links in the navbar. All relative links should work fine there. The problem with absolute links is only with shared components. The Nginx server is needed only for testing the whole setup and mapping all the different ports to a single one.
+   ```bash
+   npm login --scope=@apify-packages --registry=https://npm.pkg.github.com --auth-type=legacy
+   ```
 
-Clone all the repositories, and start the Docusaurus instances in them.
+   Read [#909](https://github.com/apify/apify-docs/issues/909) if you want to understand why this is needed.
+3. Run `npm install`.
+4. Run `npm start`. The website should open at <http://localhost:3000>.
+
+This is sufficient to work on the main content, i.e. the Academy and Platform docs.
+If you want to also work on the other parts of the docs, you may want to first join them all together using Nginx.
```

Comment:
nit: we dont use prose wrap in markdown, its 2024 and editors can wrap things for us properly if we want them to do so (and its irrelevant for the rendered version). this is actually making things worse when reviewing the git diff (in general, its not relevant to this particular place).

at some point i'd like to add some code formatter that will deal with this automatically, we actually have/had a prettier setup here but it was more of a one time thing during the big migration. maybe we could prioritize that since there are now two people working primarily on the docs, it should be simple to revive. we are experimenting with biome which is super fast so it would be even suitable to have that enable on save

Reply:
I do it for [better writing](https://sive.rs/1s), not so much for better git diffs. I'll do whatever you folks prefer, because as you say, it's two different things how I type characters to the file vs how they're committed to git.

Two people is a low number, I think we can agree on a convention just fine without tools policing us. But from what I've seen just for the past week, there's many people contributing to docs. If I'm the only one doing this tho, I still think it's easier to just tell me not to do it 😀

If you ever set up automation, I'll look over your shoulder 😀 I have Prettier on junior.guru, but I had to turn off formatting for many Markdown documents, because I have them mixed with Jinja2 etc., and that isn't something Prettier would understand.

Reply:
Fixed in [35f8595](https://github.com/apify/apify-docs/pull/937/commits/35f8595dc454d95b5f5b06bc2af4eea894775c9a)